src/Surfnet/StepupMiddlewareClientBundle/Configuration/Command/AddRaLocationCommand.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+/**
+ * Copyright 2014 SURFnet bv
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Surfnet\StepupMiddlewareClientBundle\Configuration\Command;
+
+use Surfnet\StepupMiddlewareClientBundle\Command\AbstractCommand;
+use Symfony\Component\Validator\Constraints as Assert;
+
+class AddRaLocationCommand extends AbstractCommand
+{
+    /**
+     * @Assert\NotBlank()
+     * @Assert\Type(type="string")
+     *
+     * @var string
+     */
+    public $id;
+
+    /**
+     * @Assert\NotBlank()
+     * @Assert\Type(type="string")
+     *
+     * @var string
+     */
+    public $institution;
+
+    /**
+     * @Assert\NotBlank()
+     * @Assert\Type(type="string")
+     *
+     * @var string
+     */
+    public $name;
+
+    /**
+     * @Assert\NotBlank()
+     * @Assert\Type(type="string")
+     *
+     * @var string
+     */
+    public $location;
+
+    /**
+     * @Assert\NotBlank()
```

Comment:
`$contactInformation` is allowed to be a blank string. This Assertion is not present on the Command or the Value Object in Middleware.